homeassistant/components/overkiz/sensor.py

```diff
@@ -413,6 +413,22 @@ class OverkizSensorDescription(SensorEntityDescription):
         options=["open", "tilt", "closed"],
         translation_key="three_way_handle_direction",
     ),
+    # Hitachi air to air heatpump outdoor temperature sensors (HLRRWIFI protocol)
+    OverkizSensorDescription(
+        key=OverkizState.HLRRWIFI_OUTDOOR_TEMPERATURE,
+        name="Outdoor temperature",
```

Comment:
@iMicknl the integration does not support translated entities, right?

Reply:
Indeed! It is on the list to add this in the future, however at the moment none of the entities support translation.

Reply:
Ok, I just wanted to make sure before merging 👍.